### PR TITLE
[WIP] Version 0.2.0

### PR DESCRIPTION
**Pull request by donnielrt**
Status: **WIP**
Reviewers: @fractaltheory @haroldtreen @jnurse @Helen-Mobify 

Preparing 0.2.0 release with @jnurse and @Helen-Mobify's changes.

## Todos:
- [ ] Engineer +1
- [x] Merge Jen's PR
- [x] Merge Helen's PR
- [x] Update readme and build files
- [ ] Release prep

### Feedback:
_none so far_

## Related PRs
- [x] https://github.com/mobify/hijax/pull/8
- [x] https://github.com/mobify/hijax/pull/11

## How to test
- Checkout this repo
- Ensure tests pass
- Ensure that new features work as expected
- Break features and ensure tests correctly detect failures

### CODE REVIEW

File: src/hijacker.js [hunk 1]
```diff
@@ -171,5 +171,27 @@
...
     };
 
+    // Removes the callback specified, or all callbacks for the method if no callback is given
+    Hijacker.prototype.removeListener = function(method, cb) {
+        var listeners;
+        if (!(method in this.callbacks)) {
+            throw (method + ' listener does not exist!');
+        }
+
+        listeners = this.callbacks[method];
```
Comment (donnielrt) [inline src/hijacker.js]: We can probably simplify this logic

**Comment by donnielrt** [comment]
@jnurse can you check to see if I've got your changes in correctly, and my tweaks make sense? Thanks!

**Comment by donnielrt** [comment]
@Helen-Mobify can you confirm that I've pulled in your changes correctly? Thanks!

File: README.md [hunk 2]
```diff
@@ -70,10 +79,24 @@ url/truth function and the callbacks for the events to be proxied:
...
 Additional listeners can be set like so:
 
+    hijax.addListener(<name>, <event>, <callback>);
+
+    // Eg:
     hijax.addListener('proxy1', 'complete', function(data, xhr) {
         console.log(this.name, 'Another listener.');
     });
 
+You can receive listeners on a given instance like so:
```
Comment (jnurse) [inline README.md]: This should probably say "You can remove listeners.."

**Comment by jnurse** [comment]
:+1:  Looks good, thanks @donnielrt

**Comment by Helen-Mobify** [comment]
@donnielrt :+1: The pulled in change for additional xhr param being passed back to callback function looks good!

File: README.md (hunk 2 above)
Comment (donnielrt) [inline README.md]: Fixed

**Pull request merged**